sspl_test/rabbitmq/rabbitmq_ingress_processor_tests.py

```diff
@@ -29,14 +29,14 @@
 from jsonschema import validate
 
 from pika import exceptions

```

Comment:
unused import.

Reply:
It is not an unused import. We are using it online no 98, 179, and 179. in the same file.